xml/System.Text.Encodings.Web/TextEncoder.xml

```diff
@@ -168,11 +169,11 @@
         <Parameter Name="textLength" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="text">To be added.</param>
-        <param name="textLength">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="text">The text buffer to search.</param>
+        <param name="textLength">The number of characters in the <paramref name="text"/>.</param>
+        <summary>Finds index of the first character that needs to be encoded.</summary>
+        <returns>Index of the first character that needs to be encoded.</returns>
+        <remarks>This method is seldom called directly. It's used by higher level helper APIs.</remarks>
```

Comment:
I'd probably change this to be more actionable to the customer. Something like You don't usually call this method directly.